stories/css-shadow-variable-clash.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@compiled/react';
+
+export default {
+  title: 'css prop/variable clash',
+};
+
+const isPrimary = true;
+const Component1 = styled.p`
```

Comment:
nit
I think this could be named just `Component`

Reply:
Updated the component name